requirements.pinned.txt

```diff
@@ -69,42 +75,42 @@ idna==2.6                 # via cryptography, requests
 idutils==1.0.0
 infinity==1.4             # via intervals
 intervals==0.8.1          # via wtforms-components
-invenio-access==1.0.0b1
-invenio-accounts==1.0.0b12
-invenio-admin==1.0.0b4
-invenio-assets==1.0.0b7
-invenio-base==1.0.0b1
-invenio-cache==1.0.0b1
-invenio-celery==1.0.0b3
+invenio-access==1.0.0
+invenio-accounts==1.0.0
+invenio-admin==1.0.0
+invenio-assets==1.0.0
+invenio-base==1.0.0
+invenio-cache==1.0.0
+invenio-celery==1.0.0
 invenio-communities==1.0.0a19
-invenio-config==1.0.0b3
-invenio-db[postgresql,versioning]==1.0.0b9
-invenio-files-rest==1.0.0a22  # via invenio-communities, invenio-records-files
-invenio-formatter[badges]==1.0.0b3
-invenio-i18n==1.0.0b4     # via invenio-accounts, invenio-theme
-invenio-indexer==1.0.0b1
-invenio-jsonschemas==1.0.0a7
-invenio-logging==1.0.0b3
-invenio-mail==1.0.0b1
-invenio-oaiserver==1.0.0b2
-invenio-oauth2server==1.0.0b4
-invenio-oauthclient==1.0.0b5
+invenio-config==1.0.0
+invenio-db[postgresql,versioning]==1.0.0
+invenio-formatter[badges]==1.0.0
+invenio-i18n==1.0.0     # via invenio-accounts, invenio-theme
+invenio-indexer==1.0.0
+invenio-jsonschemas==1.0.0
+invenio-logging==1.0.0
+invenio-mail==1.0.0
+invenio-oaiserver==1.0.0
+invenio-oauth2server==1.0.0
+invenio-oauthclient==1.0.0
 invenio-opendefinition==1.0.0a7
 invenio-pages==1.0.0a4
-invenio-pidstore==1.0.0b2
+invenio-pidstore==1.0.0
 invenio-previewer==1.0.0a11
-invenio-query-parser==0.6.0
 invenio-records-files==1.0.0a10
-invenio-records-rest==1.0.0b6
-invenio-records-ui==1.0.0b2
-invenio-records[postgresql]==1.0.0b4
-invenio-rest[cors]==1.0.0b2
-invenio-search-ui==1.0.0a9
-invenio-search[elasticsearch2]==1.0.0b4
+invenio-records-rest==1.0.1
+invenio-records-ui==1.0.0
+invenio-records[postgresql]==1.0.0
+invenio-rest[cors]==1.0.0
```

Comment:
I think we removed [cors], it is by default now... well, we kept for backward compatibility, but you could remove it

Reply:
oh I forgot to remove it from the pinned cause I have already removed it from `setup.py`. I will do it!